README.md

```diff
@@ -2,3 +2,89 @@
 
 Pipelayer is a framework that provides a command-line tool and Python API
 to manage [AWS Data Pipeline](http://aws.amazon.com/datapipeline/) jobs from flat files.
+Simple uses it as a cron-like job scheduler.
+
+## Overview
+
+Pipelayer aims to ease the task of scheduling jobs by defining very simple
+pipelines which are little more than an execution schedule, offloading
+most of the execution logic to files in S3.
+Pipelayer uses Data Pipeline's concept of [data staging](http://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/dp-concepts-staging.html) to pull input files from S3 at the beginning of execution and to upload output files back to S3 at the end of execution.
+
+If you follow Pipelayer's directory structure, all of your pipeline logic
+can live in version-controlled flat files. The included command-line interface
+gives you simple commands to validate your pipeline definitions, upload
+task definitions to S3, and activate your pipelines.
+
+## Installation
+
+Pipelayer is (will be...) available from PyPI:
```

Comment:
Sounds like Jake's experience is to use personal accounts for clojars, etc. I'll plan to use my own PyPI account.